swagger/sales/coupon.json

```diff
@@ -0,0 +1,124 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+      "title": "Sales",
+      "description": "An set of endpoints to manage Coupons and other sales data in the inTandem platform",
+      "version": "3.0",
+      "contact": {}
+  },
+  "tags": [],
+  "servers": [
+      {
+          "url": "https://api.vcita.biz",
+          "description": "API Gateway server"
+      }
+  ],
+  "components": {
+      "schemas": {
+          "Response": {
+              "type": "object",
+              "properties": {
+                  "success": {
+                      "type": "boolean"
+                  },
+                  "data": {
+                      "type": "object"
+                  }
+              },
+              "required": [
+                  "success",
+                  "data"
+              ]
+          },
+          "Object": {
+              "type": "object",
+              "properties": {}
+          },
+          "CouponsList": {
+            "type": "object",
+            "properties": {
+                "coupons": {
+                    "description": "List of Coupons",
+                    "type": "array",
+                    "items": {
+                        "$ref": "https://vcita.github.io/developers-hub/entities/sales/Coupon.json"
+                    }
+                }
+            }
+          }
+      }
+  },
+  "paths": {
+      "/v3/sales/coupons": {
+          "get": {
+              "summary": "Get all Coupons",
+              "description": "Get a list of Coupons - Available for **Business Tokens**",
+              "parameters": [
+                {
+                  "name": "filter",
+                  "in": "query",
+                  "schema": {  
+                    "type": "string" 
+                  },
+                  "description": "Filter parameter filter[by_status] = active / expiered / scheduled / still_available",
+                  "required": false
```

Comment:
we do not support filter anymore. if you enable filtering please state them explicitly

Reply:
Do you mean I need a parameter for by_status?